hoot-core/src/main/cpp/hoot/core/algorithms/splitter/DualWaySplitter.cpp

```diff
@@ -82,8 +82,7 @@ HOOT_FACTORY_REGISTER(OsmMapOperation, DualWaySplitter)
 
 DualWaySplitter::DualWaySplitter()
 { 
-  ConfigOptions opts = ConfigOptions();
-  if (opts.getDualWaySplitterDrivingSideDefaultValue().toLower() == "left")
+  if (ConfigOptions::getDualWaySplitterDrivingSideDefaultValue().toLower() == "left")
```

Comment:
`ConfigOptions` functions `get*DefaultValue()` are static and should be called statically and not from a specific instance.